src/plugins/guided_onboarding/public/constants/guides_config/security.ts

```diff
@@ -34,16 +43,27 @@ export const securityConfig: GuideConfig = {
     },
     {
       id: 'rules',
-      title: 'Turn on rules',
+      title: i18n.translate('guidedOnboarding.securityGuide.rulesStep.title', {
+        defaultMessage: 'Turn on rules',
+      }),
       descriptionList: [
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-        'Nullam ligula enim, malesuada a finibus vel, cursus sed risus.',
-        'Vivamus pretium, elit dictum lacinia aliquet, libero nibh dictum enim, a rhoncus leo magna in sapien.',
+        i18n.translate('guidedOnboarding.securityGuide.rulesStep.description1', {
+          defaultMessage: 'Load the prebuilt rules.',
+        }),
+        i18n.translate('guidedOnboarding.securityGuide.rulesStep.description2', {
+          defaultMessage: 'Select the rules that you want.',
+        }),
       ],
       manualCompletion: {
-        title: 'Manual completion step title',
-        description:
-          'Mark the step complete by opening the panel and clicking the button "Mark done"',
+        title: i18n.translate('guidedOnboarding.securityGuide.rulesStep.manualCompletion.title', {
+          defaultMessage: 'Continue with the tour',
+        }),
+        description: i18n.translate(
+          'guidedOnboarding.securityGuide.rulesStep.manualCompletion.description',
+          {
+            defaultMessage: 'When you’ve enabled the rules you want continue...',
```

Comment:
```suggestion
            defaultMessage: 'After you’ve enabled the rules you want, click here to continue.',
```
I think this might be close to what you originally had - now that I see it in the screenshot, this is going to make more sense.